x/claim/keeper/invariants_test.go

```diff
@@ -13,6 +13,45 @@ import (
 	"github.com/ignite/modules/x/claim/types"
 )
 
+func TestClaimRecordInvariant(t *testing.T) {
+	t.Run("should not break with valid state", func(t *testing.T) {
+		ctx, tk, _ := testkeeper.NewTestSetup(t)
+
+		tk.ClaimKeeper.SetMission(ctx, types.Mission{
+			MissionID:   10,
+			Description: "test mission",
+			Weight:      sdk.NewDec(100),
+		})
+		tk.ClaimKeeper.SetClaimRecord(ctx, types.ClaimRecord{
+			Address:           sample.Address(r),
+			Claimable:         sdkmath.NewInt(10),
+			CompletedMissions: []uint64{10},
```

Comment:
I would add another completed mission since completed bu not claimed mission this should not break the invariant